### ISSUE TITLE
"No feed found."

### ISSUE THREAD

**Issue by fnkr** [feedbin/feedbin#136]
# The problem
I got feedbin working so far but I cannot subscribe to feeds.
If I try to subscribe to a feed I get
```
No feed found.
```

# The start script
## Feedbin
```
# Mail
export SMTP_ADDRESS=...
export SMTP_USERNAME=...
export SMTP_PASSWORD=...

# Postgres
export POSTGRES_HOST=localhost
export POSTGRES_USERNAME=feedbin
export POSTGRES_PASSWORD=...
export POSTGRES_DATABASE=feedbin
export POSTGRES_PORT=5432
export DATABASE_URL=postgres://$POSTGRES_USERNAME:$POSTGRES_PASSWORD@$POSTGRES_HOST:$POSTGRES_PORT/$POSTGRES_DATABASE

# Redis
export REDIS_HOST=localhost
export REDIS_PORT=6379
export REDIS_URL=redis://$REDIS_HOST:$REDIS_PORT

# Rails
#export RAILS_ENV=production
export RAILS_ENV=development
export SECRET_KEY_BASE=...

# Path
source /usr/local/rvm/scripts/rvm
rvm use 2.1.4

# Start
if [ "$1" = "foreman" ]; then
    exec bundle exec foreman start
elif [ "$1" = "rackup" ]; then
    exec bundle exec rackup
fi
```
## Feedbin refresher
```
# Redis
export REDIS_HOST=localhost
export REDIS_PORT=6379
export REDIS_URL=redis://$REDIS_HOST:$REDIS_PORT

# Path
source /usr/local/rvm/scripts/rvm
rvm use 2.2.0

# Start
exec bundle exec foreman start
```

# The logs - after 2 minutes and 1 subscribe request
## Foreman
```
feedbin@h7:~/feedbin$ ./run foreman
Using /usr/local/rvm/gems/ruby-2.1.4
21:54:11 worker.1      | started with pid 8877
21:54:11 worker_slow.1 | started with pid 8879
21:54:11 clock.1       | started with pid 8880
21:54:15 clock.1       | I, [2015-03-06T21:54:15.579952 #8885]  INFO -- : Starting clock for 2 events: [ clockwork.frequent clockwork.daily ]
21:54:15 clock.1       | I, [2015-03-06T21:54:15.592437 #8885]  INFO -- : Triggering 'clockwork.frequent'
21:54:16 worker_slow.1 | 2015-03-06T20:54:16.023Z 8881 TID-3dynw INFO: Running in ruby 2.1.4p265 (2014-10-27 revision 48166) [x86_64-linux]
21:54:16 worker_slow.1 | 2015-03-06T20:54:16.024Z 8881 TID-3dynw INFO: See LICENSE and the LGPL-3.0 for licensing details.
21:54:16 worker_slow.1 | 2015-03-06T20:54:16.024Z 8881 TID-3dynw INFO: Upgrade to Sidekiq Pro for more features and support: http://sidekiq.org/pro
21:54:16 worker_slow.1 | 2015-03-06T20:54:16.024Z 8881 TID-3dynw INFO: Starting processing, hit Ctrl-C to stop
21:54:16 worker_slow.1 | 2015-03-06T20:54:16.027Z 8881 TID-141zt0 INFO: Booting Sidekiq 3.3.0 with redis options {:url=>"redis://localhost:6379"}
21:54:16 worker.1      | 2015-03-06T20:54:16.480Z 8878 TID-buf8g INFO: Running in ruby 2.1.4p265 (2014-10-27 revision 48166) [x86_64-linux]
21:54:16 worker.1      | 2015-03-06T20:54:16.481Z 8878 TID-buf8g INFO: See LICENSE and the LGPL-3.0 for licensing details.
21:54:16 worker.1      | 2015-03-06T20:54:16.481Z 8878 TID-buf8g INFO: Upgrade to Sidekiq Pro for more features and support: http://sidekiq.org/pro
21:54:16 worker.1      | 2015-03-06T20:54:16.481Z 8878 TID-buf8g INFO: Starting processing, hit Ctrl-C to stop
21:54:16 worker.1      | 2015-03-06T20:54:16.568Z 8878 TID-bsuzg INFO: Booting Sidekiq 3.3.0 with redis options {:url=>"redis://localhost:6379"}
21:54:16 worker.1      | 2015-03-06T20:54:16.572Z 8878 TID-gs15k FeedRefresherScheduler JID-16b32ebd33529e103295d467 INFO: start
21:54:16 worker.1      | 2015-03-06T20:54:16.574Z 8878 TID-flmh0 SendStats JID-9b46736abe4d03d580d360ae INFO: start
21:54:16 worker.1      | 2015-03-06T20:54:16.575Z 8878 TID-flmh0 SendStats JID-9b46736abe4d03d580d360ae INFO: done: 0.0 sec
21:54:16 worker.1      | 2015-03-06T20:54:16.605Z 8878 TID-gs15k FeedRefresherScheduler JID-16b32ebd33529e103295d467 INFO: done: 0.033 sec
21:55:15 clock.1       | I, [2015-03-06T21:55:15.621254 #8885]  INFO -- : Triggering 'clockwork.frequent'
21:55:15 worker.1      | 2015-03-06T20:55:15.624Z 8878 TID-flmh0 SendStats JID-6cd5a83b4c9189796df30359 INFO: start
21:55:15 worker.1      | 2015-03-06T20:55:15.624Z 8878 TID-flmh0 SendStats JID-6cd5a83b4c9189796df30359 INFO: done: 0.0 sec
21:55:15 worker.1      | 2015-03-06T20:55:15.625Z 8878 TID-gs15k FeedRefresherScheduler JID-0dd2dfeb960b99073c55c622 INFO: start
21:55:15 worker.1      | 2015-03-06T20:55:15.628Z 8878 TID-gs15k FeedRefresherScheduler JID-0dd2dfeb960b99073c55c622 INFO: done: 0.003 sec
21:56:15 clock.1       | I, [2015-03-06T21:56:15.645883 #8885]  INFO -- : Triggering 'clockwork.frequent'
21:56:15 worker.1      | 2015-03-06T20:56:15.650Z 8878 TID-gs15k FeedRefresherScheduler JID-5f71e0f3f7854841eb854ee2 INFO: start
21:56:15 worker.1      | 2015-03-06T20:56:15.651Z 8878 TID-flmh0 SendStats JID-715fa8fc88d84f6f586b60dd INFO: start
21:56:15 worker.1      | 2015-03-06T20:56:15.651Z 8878 TID-flmh0 SendStats JID-715fa8fc88d84f6f586b60dd INFO: done: 0.0 sec
21:56:15 worker.1      | 2015-03-06T20:56:15.656Z 8878 TID-gs15k FeedRefresherScheduler JID-5f71e0f3f7854841eb854ee2 INFO: done: 0.006 sec
```
## Rackup
```
feedbin@h7:~/feedbin$ ./run rackup
Using /usr/local/rvm/gems/ruby-2.1.4
[2015-03-06 21:54:26] INFO  WEBrick 1.3.1
[2015-03-06 21:54:26] INFO  ruby 2.1.4 (2014-10-27) [x86_64-linux]
[2015-03-06 21:54:26] INFO  WEBrick::HTTPServer#start: pid=8925 port=9292
46.165.246.199 - - [06/Mar/2015 21:56:07] "POST /subscriptions HTTP/1.1" 200 - 0.2454
```
## Refresher
```
feedbin@h7:~/refresher$ ./run
Using /usr/local/rvm/gems/ruby-2.2.0
21:54:44 worker.1 | started with pid 9428
21:54:45 worker.1 | 2015-03-06T20:54:45.152Z 9429 TID-9grto INFO: Running in ruby 2.2.0p0 (2014-12-25 revision 49005) [x86_64-linux]
21:54:45 worker.1 | 2015-03-06T20:54:45.153Z 9429 TID-9grto INFO: See LICENSE and the LGPL-3.0 for licensing details.
21:54:45 worker.1 | 2015-03-06T20:54:45.153Z 9429 TID-9grto INFO: Upgrade to Sidekiq Pro for more features and support: http://sidekiq.org/pro
21:54:45 worker.1 | 2015-03-06T20:54:45.153Z 9429 TID-9grto INFO: Starting processing, hit Ctrl-C to stop
21:54:45 worker.1 | 2015-03-06T20:54:45.170Z 9429 TID-dejzo INFO: Booting Sidekiq 3.3.2 with redis options {:url=>"redis://localhost:6379"}
```

# The environment
- [config/database.yaml](https://gist.github.com/fnkr/bfd79c4b32c55ed76643)
- Postgresql 9.4 [installed, `db:setup` ran successfully, started, and it looks like feedbin is connected to it, I also tried recreating the database (`db:drop` followed by `db:setup`)](https://gist.github.com/fnkr/e6534b4ba2bb3e13b39b)
- Redis 2.8 [installed, running, and it looks like feedbin is connected to it](https://gist.github.com/fnkr/da14c9428b2df6859e32)
- Elasticsearch 1.4.4: [installed, running](https://gist.github.com/fnkr/f12300122e514e520d5a)

# The code
[app/views/subscriptions/create.js.erb#L22](https://github.com/feedbin/feedbin/blob/0eb9c3dc669af584771bfe01133bbba89aee322c/app/views/subscriptions/create.js.erb#L22)
[app/controllers/subscriptions_controller.rb#L25](https://github.com/feedbin/feedbin/blob/0eb9c3dc669af584771bfe01133bbba89aee322c/app/controllers/subscriptions_controller.rb#L25)
[app/models/feed_fetcher.rb#L14](https://github.com/feedbin/feedbin/blob/0eb9c3dc669af584771bfe01133bbba89aee322c/app/models/feed_fetcher.rb#L14)

---

I have no idea what to do, please help!

**Comment by benubois** [comment]
Hi @fnkr,

Looks like you're most of the way there. Feedbin actually requires elasticsearch < 1.0. Maybe try 0.90.1.

The subscriptions controller swallows a lot of errors due to the number of things that can go wrong. However this can cause issues if there's something misconfigured. Try updating the [`SubscriptionsController#create`](https://github.com/feedbin/feedbin/blob/0eb9c3dc669af584771bfe01133bbba89aee322c/app/controllers/subscriptions_controller.rb#L25-L62) method to remove the `rescue` block so you will see the error you're getting more easily. This would look like:

```ruby
def create
  @user = current_user

  feeds = [*params[:subscription][:feeds][:feed_url]]
  site_url = params[:subscription][:site_url]
  @results = { success: [], options: [], failed: [] }

  feeds.each do |feed|
    result = FeedFetcher.new(feed, site_url).create_feed!
    if result.feed
      @user.safe_subscribe(result.feed)
      @results[:success].push(result.feed)
    elsif result.feed_options.any?
      @results[:options].push(result.feed_options)
    else
      @results[:failed].push(feed)
    end
  end

  if @results[:success].any?
    session[:favicon_complete] = false
    @mark_selected = true
    @click_feed = @results[:success].first.id
    @favicon_hash = UpdateFaviconHash.new.get_hash(@user)
    get_feeds_list
  end

  respond_to do |format|
    format.js
  end

end

```

Please let me know what you find.

Thanks!

**Comment by fnkr** [comment]
Thanks for your reply!

I purged Elasticsearch and [installed the latest Elasticsearch 0.90 (0.90.13)](https://gist.github.com/fnkr/06f393cfe950025b32a8).

I updated the `feedbin/app/controllers/subscriptions_controller.rb` and now the background request to `/subscriptions` returns a HTTP Error 422 and this stack trace: https://cdn.rawgit.com/fnkr/9e75186aeb98e685ca4f/raw/4aa55085475232e5d5b1f92afe3e618ca300f233/gistfile1.html

**Comment by fnkr** [comment]
Thanks for your reply!

I purged Elasticsearch and [installed the latest Elasticsearch 0.90 (0.90.13)](https://gist.github.com/fnkr/06f393cfe950025b32a8).

I updated the `feedbin/app/controllers/subscriptions_controller.rb` and now the background request to `/subscriptions` returns a HTTP Error 422 and a trace log:

```
ActiveRecord::RecordNotSaved in SubscriptionsController#create

Application Trace
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/persistence.rb:125:in `save!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/validations.rb:57:in `save!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/attribute_methods/dirty.rb:29:in `save!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:273:in `block in save!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:329:in `block in with_transaction_returning_status'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/connection_adapters/abstract/database_statements.rb:199:in `transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:208:in `transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:326:in `with_transaction_returning_status'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:273:in `save!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/has_many_association.rb:38:in `insert_record'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:462:in `block (2 levels) in _create_record'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:389:in `add_to_target'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:460:in `block in _create_record'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:168:in `block in transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/connection_adapters/abstract/database_statements.rb:201:in `block in transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/connection_adapters/abstract/database_statements.rb:209:in `within_new_transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/connection_adapters/abstract/database_statements.rb:201:in `transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/transactions.rb:208:in `transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:167:in `transaction'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:459:in `_create_record'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_association.rb:141:in `create!'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/associations/collection_proxy.rb:301:in `create!'
app/models/user.rb:241:in `subscribe!'
app/models/user.rb:248:in `safe_subscribe'
app/controllers/subscriptions_controller.rb:35:in `block in create'
app/controllers/subscriptions_controller.rb:32:in `each'
app/controllers/subscriptions_controller.rb:32:in `create'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/implicit_render.rb:4:in `send_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/abstract_controller/base.rb:189:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/rendering.rb:10:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/abstract_controller/callbacks.rb:20:in `block in process_action'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:113:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:113:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:229:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:229:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:166:in `block in halting'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:86:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:86:in `run_callbacks'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/abstract_controller/callbacks.rb:19:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/rescue.rb:29:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/instrumentation.rb:31:in `block in process_action'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/notifications.rb:159:in `block in instrument'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/notifications.rb:159:in `instrument'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/instrumentation.rb:30:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/params_wrapper.rb:250:in `process_action'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/railties/controller_runtime.rb:18:in `process_action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/abstract_controller/base.rb:136:in `process'
vendor/bundle/ruby/2.1.0/gems/actionview-4.1.7/lib/action_view/rendering.rb:30:in `process'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal.rb:196:in `dispatch'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_controller/metal.rb:232:in `block in action'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/routing/route_set.rb:82:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/routing/route_set.rb:82:in `dispatch'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/routing/route_set.rb:50:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/journey/router.rb:73:in `block in call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/journey/router.rb:59:in `each'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/journey/router.rb:59:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/routing/route_set.rb:678:in `call'
lib/basic_authentication.rb:10:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/content_length.rb:14:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/etag.rb:23:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/conditionalget.rb:35:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/head.rb:11:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/params_parser.rb:27:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/flash.rb:254:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/session/abstract/id.rb:225:in `context'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/session/abstract/id.rb:220:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/cookies.rb:560:in `call'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/query_cache.rb:36:in `call'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/connection_adapters/abstract/connection_pool.rb:621:in `call'
vendor/bundle/ruby/2.1.0/gems/activerecord-4.1.7/lib/active_record/migration.rb:380:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/callbacks.rb:82:in `run_callbacks'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/callbacks.rb:27:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/reloader.rb:73:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/remote_ip.rb:76:in `call'
vendor/bundle/ruby/2.1.0/gems/better_errors-2.0.0/lib/better_errors/middleware.rb:59:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/rack/logger.rb:38:in `call_app'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/rack/logger.rb:20:in `block in call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/tagged_logging.rb:68:in `block in tagged'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/tagged_logging.rb:26:in `tagged'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/tagged_logging.rb:68:in `tagged'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/rack/logger.rb:20:in `call'
vendor/bundle/ruby/2.1.0/gems/quiet_assets-1.0.3/lib/quiet_assets.rb:23:in `call_with_quiet_assets'
vendor/bundle/ruby/2.1.0/gems/silencer-0.6.0/lib/silencer/logger.rb:17:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/request_id.rb:21:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/methodoverride.rb:21:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/runtime.rb:17:in `call'
vendor/bundle/ruby/2.1.0/gems/activesupport-4.1.7/lib/active_support/cache/strategy/local_cache_middleware.rb:26:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/lock.rb:17:in `call'
vendor/bundle/ruby/2.1.0/gems/actionpack-4.1.7/lib/action_dispatch/middleware/static.rb:84:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/sendfile.rb:112:in `call'
vendor/bundle/ruby/2.1.0/gems/honeybadger-1.16.5/lib/honeybadger/rack/error_notifier.rb:43:in `call'
vendor/bundle/ruby/2.1.0/gems/honeybadger-1.16.5/lib/honeybadger/rack/user_feedback.rb:24:in `call'
vendor/bundle/ruby/2.1.0/gems/honeybadger-1.16.5/lib/honeybadger/rack/user_informer.rb:13:in `call'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/engine.rb:514:in `call'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/application.rb:144:in `call'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/railtie.rb:194:in `public_send'
vendor/bundle/ruby/2.1.0/gems/railties-4.1.7/lib/rails/railtie.rb:194:in `method_missing'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/lint.rb:49:in `_call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/lint.rb:37:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/showexceptions.rb:24:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/commonlogger.rb:33:in `call'
vendor/bundle/ruby/2.1.0/gems/sinatra-1.4.5/lib/sinatra/base.rb:217:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/chunked.rb:43:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/content_length.rb:14:in `call'
vendor/bundle/ruby/2.1.0/gems/rack-1.5.2/lib/rack/handler/webrick.rb:60:in `service'
/usr/local/rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/webrick/httpserver.rb:138:in `service'
/usr/local/rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/webrick/httpserver.rb:94:in `run'
/usr/local/rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/webrick/server.rb:295:in `block in start_thread'
```

- https://gist.github.com/fnkr/7ac014aaf5b10f43de96
- https://github.com/rails/rails/blob/master/activerecord/lib/active_record/persistence.rb#L125
- https://github.com/rails/rails/blob/4591b0fc041454f4ba4a83629b9bbca2a851969c/activerecord/lib/active_record/callbacks.rb#L301

**Comment by fnkr** [comment]
I had the idea to enable the postgres log and subscribe to a feed.
https://gist.github.com/fnkr/503c09fced55b0797c0f

After that I checked the database and it seems like its just the entry in the `subscriptions` that fails.
If I add it manually it works and I can access the feed from the web ui.
https://gist.github.com/fnkr/a478384ed67afd2f2224

The strange thing is that there is no `insert` statement for the `subscriptions` table in the log - feedbin doesnt even try to insert something.

**Comment by fnkr** [comment]
Fixed the bug by installing memcached and setting the corresponding environment variable.

**Comment by benubois** [comment]
Glad it’s working!

**Comment by fnkr** [comment]
In fact, the problem wasn't that memcached wasn't installed. The environment variable was set to `localhost:11211` which resolves to `::1:11211` on my machine, however memcached is, at least if installed from the Debian Wheezy  repository,  configured to listen on `127.0.0.1:11211` only by default. There are three solutions: 1) update the memcached configuration to listen on the ipv6 loopback too 2) change `localhost` to `127.0.0.1` 3) update the hosts file so that `localhost` resolves to `127.0.0.1` instead (be careful this could break other things).
